### DISCUSSION TITLE
Are there any plans for throttling rules?

### DISCUSSION THREAD

**Discussion by jonathonbyrdziak** [postalserver/postal#1767] [Feature suggestions]
As you know a major function of managing reputation globally is to manage reputation with each receiving service individually. Most enterprise grade MTAs offer throttling rules, does Postal have any plans for implementing throttling rules?

**Comment by willpower232** [comment]
I don't think so, this would not be needed unless you are hosting Postal for third parties which may or may not be trustworthy.

That said, you can implement a limit so that only X number of messages can be sent per hour within the settings already but that can cause a manual clearup to be required as the messages will still be stored by Postal to be sent manually.

**Comment by afanjul** [comment]
We are using Postal since years ago, and one of our major problems is related with the [Google Sending Rate limits](https://support.google.com/mail/answer/81126?p=UnsolicitedRateLimitError&visit_id=637756893770395839-2838446055&rd=1), even you are sending good transactional valid emails (not marketing, or spam, or mass emails), Google has some sending rate limits that you can check on the postmaster tool...

![image](https://user-images.githubusercontent.com/2046424/146946468-d140b558-f1e9-442f-97ea-582e5ec17c05.png)

So in our case, we clearly lack a Throtteling system to lower the rate of emails sent by minute, without having to manually unrelease them... (that we obviously don't do because it has non sense in our case)
